packages/logger/index.js

```diff
@@ -0,0 +1,123 @@
+const logger = require('../dd-trace/src/log')
```

Comment:
Packages should only be used for things that could be independent packages and don't depend on dd-trace, so let's move this to dd-trace. Let's also rename it since it's otherwise fairly confusing with the existing logger.

Reply:
So move it to someplace like `packages/dd-trace/src/log-writer.js`?

Reply:
If it's a specific kind of logger, it should be obvious in the name. Otherwise a "logger" makes it sound like it will just output logs to some standard output, when actually this is sending "logs" as some sort of events for our backend to use.

Reply:
Sorry I think my naming is confusing the logger that I am pulling in is the tracer logger, it was to log anything happening in the logger to the tracer output.

Reply:
I can also rename the directory of this log writer, right now it is nested in packages/logger/ which is not very descriptive